test/unit.js

```diff
@@ -212,13 +206,12 @@ test('should call provided error callback on error.', function (t) {
 });
 
 test('should call provided error callback is bad JSON error.', function (t) {
-    t.plan(3);
+    t.plan(2);
 
     var xhr = sync('read', modelStub(), {
         error: function (resp, type, error) {
             t.deepEqual(resp, {}, 'should be passed through response');
             t.equal(type, 'error', 'is string \'error\' as per jquery');
-            t.ok(error == 'Unable to parse JSON string' || error == 'Unexpected end of input', 'should be json parse message');
```

Comment:
I found another new message for this error recently, no point checking that anymore, it'd fail sooner or later